gsy_framework/utils.py

```diff
@@ -238,6 +238,11 @@ def datetime_to_string_incl_seconds(date_time: DateTime) -> str:
     return date_time.format(DATE_TIME_FORMAT_SECONDS)
 
 
+def pendulum_datetime_to_string(datetime_object: DateTime) -> str:
+    """Convert a datetime object into a string with YYYY-MM-DDTHH:mm format."""
+    return datetime_object.format(DATE_TIME_FORMAT)
```

Comment:
Minor, however I think that the `format_datetime` function could be used instead of creating a separate function here.

Reply:
[68f70c5](https://github.com/gridsingularity/gsy-framework/pull/397/commits/68f70c5b60c290d45c1332c8350e7316b1da0c5e)